pkgs/development/python-modules/uni-curses/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, ncurses, x256, setuptools }:
+
+buildPythonPackage rec {
+  pname = "unicurses";
+  version = "2.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "Uni-Curses";
+    sha256 = "sha256-Anz5YMZULdMhZ802T3tj9LOcBZZfUML1fPOJbC47nVA=";
+  };
+
+  buildInputs = [ ncurses ];
+  propagatedBuildInputs = [ x256 ];
+
+  postPatch = ''
+    substituteInPlace './unicurses/__init__.py' \
+      --replace "find_library('ncursesw')" '"${ncurses}/lib/libncursesw.so.6"' \
+      --replace "find_library('panelw')" '"${ncurses}/lib/libpanelw.so.6"'
+  '';
```

Comment:
Necessary because `ctypes.util.find_library` does not find the ncurses libraries. I couldn't find a cleaner fix, but if there is one, I'll modify it.

Reply:
> Necessary because ctypes.util.find_library does not find the ncurses libraries.

There seems to be a precedent for doing this [in other packages](https://github.com/NixOS/nixpkgs/blob/7deab23a756905fcbcc6fb13e7a27b0c0f3acdbf/pkgs/development/python-modules/pyzbar/default.nix#L26) so it should be fine, but please move this comment in to the code so it is easier to find.